src/box/sql/sqlInt.h

```diff
@@ -1014,6 +1014,8 @@ struct sql {
 					 */
 enum {
 	SQL_SeqScan = 0x00000008,
+	SQL_DEFAULT_FLAGS = SQL_EnableTrigger | SQL_AutoIndex |
+			    SQL_RecTriggers | SQL_SeqScan,
```

Comment:
Nit: Why don't you use a macro for these constants, like for other SQL flags? It'd look more consistent.

Reply:
I'm planning to remake the other flags into `enum` instead of a macro. Not now, though.

Reply:
Okay. TBH I dislike using enum for bitwise flags. Is there a good reason for that?

Reply:
Nothing special, I think. Just prefer enum over macro in such cases.